packages/api-docs-builder/buildApiUtils.ts

```diff
@@ -432,9 +432,9 @@ export function getJoyComponentInfo(filename: string): ComponentInfo {
         return null;
       }
       return {
-        name: inheritedComponent,
+        name: inheritedComponent.replace(/unstyled/i, ''),
         apiPathname: `/${inheritedComponent.match(/unstyled/i) ? 'base' : 'joy-ui'}/api/${kebabCase(
```

Comment:
We still have a check for `unstyled` here

Reply:
Let's add comments on these lines then, for future reference as well as test in at least of the component's documentation, so that we can potentially catch issues in the future.

Reply:
agree, done